doc/manual/citing_aspect.bib

```diff
@@ -713,9 +713,21 @@ @article{citron2020effects
   number={4},
   pages={e2019GC008895},
   year={2020},
-  publisher={Wiley Online Library}, 
+  publisher={Wiley Online Library},
   doi={10.1029/2019GC008895},
   url={https://doi.org/10.1029/2019GC008895}
 }
 
-
+@article{HEYN2020116358,
+	title = "Core-mantle boundary topography and its relation to the viscosity structure of the lowermost mantle",
+	journal = "Earth and Planetary Science Letters",
+	volume = "543",
+	pages = "116358",
+	year = "2020",
+	issn = "0012-821X",
+	doi = "https://doi.org/10.1016/j.epsl.2020.116358",
+	url = "http://www.sciencedirect.com/science/article/pii/S0012821X20303022",
+	author = "Björn H. Heyn and Clinton P. Conrad and Reidar G. Trønnes",
```

Comment:
Just to prevent causing issues with special  characters. Otherwise good to merge.

```suggestion
	author = "Bj{\"o}rn H. Heyn and Clinton P. Conrad and Reidar G. Tr{\o}nnes",
```